.github/workflows/ci.yml

```diff
@@ -74,7 +74,7 @@ jobs:
       - name: API Extractor Check
         run: |
```

Comment:
When CI succeeds can you confirm the `Upload azure-communication api.md files for easy access` (though should say api.json files) was successful and the api files got uploaded correctly?

Reply:
Will do. I did check that we're picking the files off the `/temp/` folder which shouldn't change.